app/Http/Controllers/TransactionsController.php

```diff
@@ -4,18 +4,42 @@
 
 namespace App\Http\Controllers;
 
-use App\Models\Transaction;
+use App\Services\BigNumber;
+use App\Services\Timestamp;
+use Carbon\Carbon;
 use Illuminate\Contracts\View\View;
-use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
 
 final class TransactionsController
 {
-    public function __invoke(Request $request, Transaction $transaction): View
+    public function __invoke(): View
     {
-        $type = $request->input('state.type', 'all');
+        $data = $this->transactionData();
 
         return view('app.transactions', [
-            'transactionTypeFilter' => $type,
+            'transactionCount' => $data['transaction_count'],
+            'volume'           => BigNumber::new($data['volume'])->toFloat(),
+            'totalFees'        => BigNumber::new($data['total_fees'])->toFloat(),
+            'averageFee'       => BigNumber::new($data['average_fee'])->toFloat(),
         ]);
     }
+
+    private function transactionData(): array
+    {
+        $data = (array) DB::connection('explorer')
```

Comment:
shall we cache this for a few minutes? shouldn't matter if the stats lag behind a little

Reply:
I'm down for that